language-service/src/yamlLanguageService.ts

```diff
@@ -102,6 +109,8 @@ export interface LanguageService {
   doResolve(completionItem: CompletionItem): Thenable<CompletionItem>;
   resetSchema(uri: string): boolean;
   doFormat(document: TextDocument, options: FormattingOptions, customTags: Array<String>): TextEdit[];
+  findNodes(document: TextDocument, doc: YAMLDocument, key: string): Thenable<YamlNodeInfo[]>;
+  getNodeInputValues(document: TextDocument, doc: YAMLDocument, position: Position): {[key: string]: string};
```

Comment:
Recommendation is to make output type of this method an actual type or class so in the future if we need to add any additional information to the return we don't need to break the compat.